test/test_helper.exs

```diff
@@ -1 +1,4 @@
+{:ok, _} = Application.ensure_all_started(:bypass)
+{:ok, _} = Application.ensure_all_started(:hackney)
+{:ok, _} = Application.ensure_all_started(:mox)
```

Comment:
This is needed because these are optional deps which aren't automatically added as `extra_applications` in the mix project's `application/0`